### ISSUE TITLE
no error when checking path that doesn't exist

### ISSUE THREAD

**Issue by DetachHead** [pylint-dev/pylint#10136]
### Bug description

```python
when running pylint on a non-existent path, no error is reported and the command exits with code 0
```

### Configuration

```ini

```

### Command used

```shell
pylint path_that_doesnt_exist
```

### Pylint output

```python
no output
```

### Expected behavior

error stating that the path does not exist

### Pylint version

```shell
pylint 3.3.2
astroid 3.3.5
Python 3.13.0 (main, Oct 16 2024, 00:33:24) [MSC v.1929 64 bit (AMD64)]
```

### OS / Environment

windows 11

### Additional dependencies

```python

```

**Comment by vasiliyk** [comment]
````shell
(.venv) vasiliy@night pylint % python -m pylint --version
pylint 4.0.0-dev0
astroid 4.0.0
Python 3.13.7 (main, Aug 14 2025, 11:12:11) [Clang 17.0.0 (clang-1700.0.13.3)]
````

````shell
(.venv) vasiliy@night pylint % python -m pylint no_file.py           

************* Module your_file.py
your_file.py:1:0: F0001: No module named your_file.py (fatal)
````

````shell
(.venv) vasiliy@night pylint % python -m pylint pylint path_that_doesnt_exist
************* Module path_that_doesnt_exist
path_that_doesnt_exist:1:0: F0001: No module named path_that_doesnt_exist (fatal)

-----------------------------------
Your code has been rated at 0.00/10
````

**Comment by Pierre-Sassoulas** [comment]
@vasiliyk what is your operating system ? I can't check on windows 11 personally but I assumed that the bug was valid because @KotlinIsland thumbs-upped the issue.

**Comment by DetachHead** [comment]
confirmed this is still occurring for me on windows 11 with the latest version of `4.0.0-dev0` from the main branch (5f0298dcf642742e8980266f6e243c760ffd0865):

```
(project) PS> pylint --version
pylint 4.0.0-dev0
astroid 4.0.0
Python 3.13.3 (main, Apr  9 2025, 04:04:49) [MSC v.1943 64 bit (AMD64)]
(project) PS> pylint path_that_doesnt_exist
(project) PS> pylint path_that_doesnt_exist.py
(project) PS>
```

let me know if you need any more info. i will test on linux when i get home later today

**Comment by DetachHead** [comment]
happens on linux as well (Linux Mint 22.2)

**Comment by Pierre-Sassoulas** [comment]
Can you show your configuration (init-hook in particular) ?

**Comment by DetachHead** [comment]
omg... i had the rule disabled, sorry!
